bash/solutii/micu_matei/compare.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Sa se scrie un fisier de comenzi, care verifica daca doua directoare sunt egale.
+# Numele celor doua directoare se transmit ca argumente in linia de comanda. Doua
+# directoare se considera ca sunt egale daca contin aceleasi subdirectoare si fisiere.
+# Se utilizeaza comanda diff.
+
+if [[ $# -ne 2 ]]; then
+    echo "Not enough params !";
+    exit;
```

Comment:
Iti recomand sa pui ` exit 1; `. Daca acest script este apelat si vrei sa folosesti exit_code, in loc de eroare(1) o sa iti apare succes(0).